src/main/java/uk/gov/ida/eventemitter/LocalEncrypter.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.ida.eventemitter;
+
+import com.google.inject.Inject;
+import org.apache.commons.codec.binary.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+
+public class LocalEncrypter implements Encrypter {
+
+    private final String key;
+    private final String initVector;
+
+    @Inject
+    public LocalEncrypter(final String key, final String initVector) {
+        this.key = key;
+        this.initVector = initVector;
+    }
+
+    @Override
+    public String encrypt(final Event event) {
+        try {
+            final IvParameterSpec iv = new IvParameterSpec(initVector.getBytes("UTF-8"));
+            final SecretKeySpec skeySpec = new SecretKeySpec(key.getBytes("UTF-8"), "AES");
+            final Cipher cipher = Cipher.getInstance("AES/CBC/PKCS5PADDING");
+            cipher.init(Cipher.ENCRYPT_MODE, skeySpec, iv);
+
+            final byte[] encryptedEvent;
+
+            try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Perhaps I'm being dim (I certainly don't know much about encryption) but this block looks a little magic to me.
Could you walk me through whats going on?

Reply:
I have changed LocalEncrypter to use JSON instead of Java Object for consistency since verify-event-service-recorder uses JSON.